views/about.ejs

```diff
@@ -3,7 +3,48 @@
 	<%- include('partials/header'); %>
 	<main class="mdl-layout__content">
 		<div class="mdl-typography--text-center clemson-esports">
-			<h1>Clemson Esports</h1>
+			<h1>About Clemson Esports</h1>
 		</div>
+		<div class="mdl-typography--text-center clemson-esports">
```

Comment:
There is too much space between this text and the header.

Reply:
Padding has been halved
![image](https://user-images.githubusercontent.com/24688719/162633892-0ce85e07-b1ec-4808-b8bf-ba452e6b8e38.png)